src/merkle/__tests__/merkle-metadata.test.ts

```diff
@@ -19,6 +19,7 @@ const TYPE_REGEX =
   /^jsnpm_@ceramicnetwork\/wasm-bloom-filter-v((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$/
 const isTypeString = (str: string) => Boolean(str.match(TYPE_REGEX))
 
+
```

Comment:
Do we need this whitespace?

Reply:
It's not required, but it was there ^_^

Reply:
I just know that in the present form it eliminates dev -> rc conflict. IDK what happens, if I remove it here.